content/rs/security/access-control/active-active.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Access control for Active-Active databases
+linkTitle: Active-Active databases
+description: Access control for Active-Active databases.
+weight: 60
+alwaysopen: false
+categories: ["RS"]
+aliases: 
+---
+
+Users, roles, and Redis ACLs are cluster-level entities. Therefore they are applied per local participating cluster and Active-Active database instance, and they are not replicated or propagated to the other participating clusters and instances.
+ACLs will be enforced according to the instance the client is connected to. The Active-Active replication mechanism will propagate all the effects of the operation.
+
+## Update admin credentials
```

Comment:
This page holds two things:
1. The above - ACL - is about data access permissions.
2. The below - admin user - is about the control plane Admin user.
I think we need to clarify it a little bit more, maybe a title for the first will do the work. They just seems connected to me visually at the moment

Reply:
Added another header.